docs/src/main/paradox/running.md

```diff
@@ -0,0 +1,71 @@
+# Running a Projection
+
+Once you have decided how you want to build your projection, the next step is to run it. Typically, you run it in a distributed fashion in order to spread the load over the different nodes in an Akka Cluster. However, it's also possible to run it as a single instance (when not clustered) or as single instance in a Cluster Singleton.
+
+## Dependencies
+
+To distribute the projection over the cluster we recommend the use of [SharderDaemonProcess](https://doc.akka.io/docs/akka/current/typed/cluster-sharded-daemon-process.html). Add the following dependency in your project if not yet using Akka Cluster Sharding:
+
+@@dependency [sbt,Maven,Gradle] {
+  group=com.typesafe.akka
+  artifact=akka-cluster-sharding-typed_$scala.binary.version$
+  version=$akka.version$
+}
+
+Akka Projections require Akka $akka.version$ or later, see @ref:[Akka version](overview.md#akka-version).
+
+For more information on using Akka Cluster consult Akka's reference on [Akka Cluster](https://doc.akka.io/docs/akka/current/typed/index-cluster.html) and [Akka Cluster Sharding](https://doc.akka.io/docs/akka/current/typed/cluster-sharding.html).
+
+## Sharded Daemon Process Example
+
+The Sharded Daemon Process can be used to distribute `n` instances of a given Projection across the cluster. Therefore, it's important that each Projection instance consumes a subset of the stream of envelopes.
+
+How the subset is created depends on the kind of source we consume. If it's a Alpakka Kafka source, this is typically done using Kafka partitions. When consuming from Akka Persistence Journal, the events must be tagged with a slice number as demonstrated in the example bellow.
+
+### Tagging Events in EventSourcedBehavior
+
+Before we can consume the events, the entity must tag the events with a slice number.
+
+Scala
+:  @@snip [ShoppingCart.scala](/examples/src/test/scala/docs/eventsourced/ShoppingCart.scala) { #imports #slicingTags }
+
+FIXME Java examples
+
+In the above example, we created a `Vector` of tags from *carts-0* to *carts-4*. Each entity instance will tag its events using one of those tags. The tag is selected based on the module of the entity id's hash code (stable identifier) and the total number of tags. As a matter of fact, this will create a journal sliced with different tags (ie: from *carts-0* to *carts-4*).
+
+We will use those tags to query the journal and create as many Projections instances, and distribute them in the cluster.
+
+### Event Sourced Provider per tag
+
+We can use @ref:[EventSourcedProvider.eventsByTag](eventsourced.md) to consume the `ShoppingCart` events.
```

Comment:
eventsourced.md#sourcedprovider-for-eventsbytag

Reply:
I have a second thought about this part. 

I think it should be 

> We can use the @ref:[EventSourcedProvider](eventsourced.md) to consume the `ShoppingCart` events.

And link to the page, not to the section. We are already show on this page who to use eventsByTag. What we want is to introduce the use to `EventSourcedProvider`, not how to use it.

Reply:
ok, sounds good